hubspot_commerce.module

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @file
+ * Hubspot_commerce.module.
+ */
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+
+const HUBSPOT_COMMERCE_LOGGER_CHANNEL = 'hubspot_commerce';
+
+/**
+ * Implements hook_help().
+ */
+function hubspot_commerce_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    case 'help.page.hubspot_commerce':
+      $output = '';
+      $output .= '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('The Hubspot Commerce module syncs data with the <a target="_blank" href=":hubspot_api">Hubspot API</a> any time a contact, order, or product is created or updated.', [':hubspot_api' => 'https://www.hubspot.com']) . '</p>';
+
+      return $output;
+  }
+}
+
+/**
+ * Implements hook_entity_update().
+ */
+function hubspot_commerce_entity_update(EntityInterface $entity) {
+  // TODO: Make this configurable.
+  $entities_to_sync = [
+    'profile',
+    'order',
+    'order_item',
+    'product_variation',
+  ];
```

Comment:
**CHANGE REQUEST**

*Application logic*

The `User` entity should be added here as well.

*Architecture*

Let's dispatch an event here that allows other modules to alter the list of entities that will be synced. Note that the entities configured to be synced *from* Drupal *to* HubSpot might not be the same as the entities synced *from* HubSpot *to* Drupal (when that feature will be implemented, so make sure you name the event appropriate, something like `commerce_hubspot.entity_sync.to` would do.